plugins/nordic-hid/fu-nordic-hid-cfg-channel.c

```diff
@@ -1684,11 +1687,8 @@ fu_nordic_hid_cfg_channel_init(FuNordicHidCfgChannel *self)
 static FuNordicHidCfgChannel *
 fu_nordic_hid_cfg_channel_new(guint8 id, FuNordicHidCfgChannel *parent)
 {
-	FuNordicHidCfgChannel *self = g_object_new(FU_TYPE_NORDIC_HID_CFG_CHANNEL,
-						   "context",
-						   fu_device_get_context(FU_DEVICE(parent)),
-						   NULL);
+	FuNordicHidCfgChannel *self =
+	    g_object_new(FU_TYPE_NORDIC_HID_CFG_CHANNEL, "proxy", parent, NULL);
```

Comment:
Should we use `FU_DEVICE(parent)` to ensure proper type here?

Reply:
I don't think it matters, although if you'd rather I'm 100% okay with it if you want to push that to the branch.